.github/workflows/pythonpackage.yml

```diff
@@ -67,15 +67,9 @@ jobs:
         python -m venv /tmp/setup_install
```

Comment:
You can go ahead and get rid of `setup.py - install`. Setuptools does not recommend installing this way. You could replace with `pip install .`. You should not need to install dependencies manually first.